packages/admin-ui/src/page/page-header-title.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import type { ComponentPropsWithRef } from 'react'
+import { createComponent, useElement, tag } from '@vtex/admin-ui-react'
+import { IconArrowLeft } from '@vtex/phosphor-icons'
+
+import { Button } from '../button'
+import { usePageHeaderContext } from './page-header-context'
+import * as style from './page.style'
+
+/**
+ * Page header title component
+ *
+ * @example
+ * import { PageHeader, PageHeaderTop, PageHeaderTitle, Stack, Tag } from "@vtex/admin-ui"
+ *
+ * <PageHeader>
+ *  <PageHeaderTop>
+ *    <PageHeaderTitle>
+ *      Product #123
+ *      <Stack direction="row" space="$m">
+ *        <Tag label="Short text" size="large" />
+ *        <Tag label="Short text" size="large" />
+ *      </Stack>
+ *    </PageHeaderTitle>
```

Comment:
I'm trying to figure out a better approach to it. One option is to do the `Stack` space behavior inside the `PageHeaderTitle` csx, [like this](https://codesandbox.io/s/reverent-dew-1kelbf?file=/src/Root.tsx).
The bad part is that this requires to pass the `Product #123` label inside a `<p>` tag

Reply:
Other option would be to have a `tags` property in the `PageHeaderTitle`, something like this:
```jsx
<PageHeaderTitle tags={[{label, tone}, {label, tone}]}>
  Product #123
</PageHeaderTitle>
```

Reply:
And the last option I can imagine is to separate the Tag from the title. It looks to me they are two different things and that's why it's being difficult to find a better solution

```
<PageHeader>
  <PageHeaderTop>
    <PageHeaderTitle>...</PageHeaderTitle>
    <PageHeaderTags>
       <Tag />
       <Tag />
     </PageHeaderTags>
     <PageHeaderActions>...</PageHeaderActions>
  </PageHeaderTop>
</PageHeader>
```

Reply:
I'm happy with the current approach, where we got a loose API. This gives us the flexibility to let this component evolve without us having to change its code. 

It occurs to me that with good documentation, our clients will figure out how to add tags on their own, with no need for magic under the hood or explicit wrappers (like PageHeaderTags)

Reply:
[I have used the second approach before](https://github.com/vtex/admin-ui/pull/267/commits/bd8e30eda8585f04f06aaed0b30aefd85390d7d6#diff-046718c6255d4ea5aea39a1e01e02196383a13731e54626c2124cfab408688b3R96), but it looks like this is an anti-pattern for our project, since we are applying compound components to most of our "complex" components